File: java/pca-model-generator/src/main/java/GraphQLSchemaType.java
import java.util.List;

public class    GraphQLSchemaType {
    public String kind;
    public String name;
    public String description;
    public List<GraphQLSchemaField> fields;
    public List<GraphQLSchemaEnum> enumValues;
    public List<GraphQLSchemaInterface> interfaces;
    public List<GraphQLSchemaTypeInfo> possibleTypes;
    public List<GraphQLSchemaField> inputFields;
}

File: net/src/Sdl.Tridion.Api.Client/GraphQLClient/Schema/GraphQLSchemaType.cs
using System.Collections.Generic;

namespace Sdl.Tridion.Api.GraphQL.Client.Schema
{
    public class GraphQLSchemaType
    {
        public string Kind { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public List<GraphQLSchemaField> Fields { get; set; }
        public List<GraphQLSchemaEnum> EnumValues { get; set; }
        public List<GraphQLSchemaInterface> Interfaces { get; set; }
        public List<GraphQLSchemaTypeInfo> PossibleTypes { get; set; }
        public List<GraphQLSchemaField> InputFields { get; set; }
    }
}
